specs/person_spec.rb

```diff
@@ -0,0 +1,47 @@
+require_relative '../person'
```

Comment:
- [ ] It's a requirement to make sure to check there is an 'it' block for each class method (at least). Kindly add this for the `add_rental` method.